README.md

```diff
@@ -436,6 +461,49 @@ const token = await stripe.createTokenWithCard(params)
 // api.sendTokenToBackend(token)
 ```
 
+### Request with external (bank) account params object (Android only at the moment)
+
+#### `createTokenWithBankAccount(params) -> Promise`
+
+Creates token based on external (bank) params.
+
+##### `params`
+
+An object with the following keys:
+
+* `routingNumber` _String_ (Required) - The routing number of this account.
+* `accountNumber` _String_ (Required) - The account number for this BankAccount.
+* `countryCode`  _String_ (Required) - The two-letter country code that this account was created in.
+* `currency`  _String_ (Required) - The currency of this account.
+* `accountHolderName` _String_ - The account holder's name.
+* `accountHolderType` _String_ - the bank account type. Can be one of: `company`|`individual`.
+* `fingerprint` _String_ - The account fingerprint.
+* `bankName` _String_ - The name of bank.
+* `last4` _String_ - The last four digits of the account number.
```

Comment:
`params` does not contain `fingerprint`, `bankName` and `last4`; `routingNumber` is not required and should be placed after required fields.